packages/apps/shopify-app-remix/src/server/authenticate/public/factory.ts

```diff
@@ -45,5 +49,5 @@ export function authenticatePublicFactory<
   authenticatePublic.checkout = authenticateCheckout;
   authenticatePublic.appProxy = authenticateAppProxy;
 
-  return authenticatePublic;
+  return authenticatePublic as AuthenticatePublic<Future>;
```

Comment:
Why do we need to cast the type here? I would think that we're returning the appropriate type given the definitions.

Reply:
We do also cast  the return above,
 I don't know if typescript was just struggling with evaluating the FF type in this scenario, but I was getting an error.